### ISSUE TITLE
Add OAEP padding sha256 support

### ISSUE THREAD

**Issue by dev4jam** [TakeScoop/SwiftyRSA#118]
Any plans to add OAEP with SHA256 support?

**Comment by ldiqual** [comment]
I don't believe apple SDKs support this out of the box but I'm happy to review any PR that would help achieve this goal.
